gitignore/go

```diff
@@ -0,0 +1,9 @@
+# Test output
+test.xml
+
+# Lint output
+lint.txt
+
+# Mac files
+.DS_Store
+._*
```

Comment:
we should include a section for ignoring build artifacts:

```
# Build artifacts
/{REPOSITORY_NAME}
{REPOSITORY_NAME}-*.zip
build.log
```

see https://github.com/companieshouse/chs-monitor-consumer/pull/15 for example in chs-monitor-consumer:

```
# Build artifacts
/chs-monitor-consumer
chs-monitor-consumer-*.zip
build.log
```

Reply:
Added